deeplearning4j/deeplearning4j-common-tests/pom.xml

```diff
@@ -76,25 +76,48 @@
         </profile>
         <profile>
             <id>nd4j-tests-cuda</id>
+            <properties>
+                <!-- Note we put these properties here because the cuda modules themselves  have the cuda version tied to the artifact name. This allows us to use variables for the testing.  -->
+                <cuda.version>11.0</cuda.version>
+                <cudnn.version>8.0</cudnn.version>
+                <javacpp-presets.cuda.version>1.5.4</javacpp-presets.cuda.version>
+                <junit.jupiter.execution.parallel.enabled>false</junit.jupiter.execution.parallel.enabled>
+                <junit.jupiter.execution.parallel.mode.default>same_thread</junit.jupiter.execution.parallel.mode.default>
+
+            </properties>
             <dependencies>
                 <dependency>
                     <groupId>org.nd4j</groupId>
                     <artifactId>nd4j-cuda-11.0</artifactId>
```

Comment:
When trying to make the cuda version dynamic, I guess you'll have to set the version here too.

Reply:
Good catch! Remember, this is mainly for unreleased versions where we're just dealing with tests. Released versions have to be static which is why we do release, change-cuda-versions.sh, release

Reply:
We are looking at test dependencies here, so it doesn't really matter for released vs. unreleased.

If this stays the same, you need to both use the change-cuda-versions.sh script and set the properties for the cuda platform redist.